### PR TITLE
Add support for stream

### LINKED ISSUE #282: New proto3 keyword: stream
https://github.com/grpc/grpc-java/blob/master/examples/src/main/proto/route_guide.proto#L51

I don't see this in the formal proto3 docs anywhere.

### PR DESCRIPTION
**Pull request by rdegnan**
Resolves #282

### CODE REVIEW

**Comment by JakeWharton** [comment]
Looks good. Can you add a test for the parsing of these new keywords? Surely we have one already for RPCs you just gotta find it.

**Comment by rdegnan** [comment]
Sure, will do.

File: wire-schema/src/main/java/com/squareup/wire/schema/internal/parser/RpcElement.java [hunk 1]
```diff
@@ -66,6 +76,8 @@ public final String toSchema() {
     Builder requestType(String requestType);
     Builder responseType(String responseType);
     Builder options(ImmutableList<OptionElement> options);
+    Builder clientStreaming(boolean clientStreaming);
```
Comment (swankjesse) [inline wire-schema/src/main/java/com/squareup/wire/schema/internal/parser/RpcElement.java]: should these be `requestStreaming` and `responseStreaming`?

Comment (rdegnan) [inline wire-schema/src/main/java/com/squareup/wire/schema/internal/parser/RpcElement.java]: They could, I was just copying the name from MethodDescriptorProto. I agree requestStreaming and responseStreaming would be clearer.

**Comment by rdegnan** [comment]
Ok, renamed fields to requestStreaming/responseStreaming and added tests.

**Comment by JakeWharton** [comment]
Thanks! LGTM. @swankjesse to review and merge.

File: wire-schema/src/main/java/com/squareup/wire/schema/Rpc.java [hunk 2]
```diff
@@ -27,14 +27,18 @@
   private final Options options;
   private ProtoType requestType;
   private ProtoType responseType;
+  private final boolean requestStreaming;
+  private final boolean responseStreaming;
 
   private Rpc(Location location, String name, String documentation, String requestType,
-      String responseType, Options options) {
+      String responseType, boolean requestStreaming, boolean responseStreaming, Options options) {
```
Comment (swankjesse) [inline wire-schema/src/main/java/com/squareup/wire/schema/Rpc.java]: thanks!

**Comment by swankjesse** [comment]
This is wonderful. Thank you. Merged manually.

**Comment by swankjesse** [comment]
https://github.com/square/wire/commit/4fb8fa40773faea1b42727d90ae11c969982a101

**Pull request closed without merging**